kube/ims-resolver/ims-resolver-deploy.yml

```diff
@@ -39,7 +39,7 @@ spec:
     spec:
       containers:
         - name: ims-resolver
-          image: quay.io/ukhomeofficedigital/ims-resolver:2eb902d215c8863e2dd1714f4c4bdd0351b87f62
+          image: quay.io/ukhomeofficedigital/ims-resolver:1b49a51cdf048eb3c693d23f54587d25074b5019
```

Comment:
Up to you but it may be useful to add the image digest from the merged pipeline of your accompanying ims-resolver as it will roll in a small fix change I've added in the last few days.